projects/core/src/configurator/commons/store/effects/configurator.effect.ts

```diff
@@ -151,6 +155,30 @@ export class ConfiguratorEffects {
     })
   );
 
+  @Effect()
+  getOverview$: Observable<
+    GetConfigurationOverviewSuccess | GetConfigurationOverviewFail
+  > = this.actions$.pipe(
+    ofType(GET_CONFIGURATION_OVERVIEW),
+    map((action: GetConfigurationOverview) => action.payload),
+    mergeMap(payload => {
+      return this.configuratorCommonsConnector
+        .getConfigurationOverview(payload.configId, payload.owner)
+        .pipe(
+          map((configuration: Configurator.Configuration) => {
+            return new GetConfigurationOverviewSuccess(configuration);
```

Comment:
Constructor of OverviewSucces changed to (owner, overview)?

Reply:
Done.